scripts/unit_test.sh

```diff
@@ -13,7 +13,7 @@ mkdir -p test_results/latest
 echo "Running unit tests and placing results/coverage in test_results/${timestamp} on host..."
 
 # Avoid the db/migrations package since it only contains test code
-PACKAGES_TO_COVER=$(go list ./... | egrep -v 'test|mock|db/migrations' | tr "\n" "," | sed -e 's/,$//g')
+PACKAGES_TO_COVER=$(go list ./... ./optout | egrep -v 'test|mock|db/migrations' | tr "\n" "," | sed -e 's/,$//g')
```

Comment:
do we need to also update the `lint` command in the makefile to include this package?